File: Programming Basics with Python/03 Conditional statements advanced/More exercises/05_Vacation.py
budget = float(input())
season = input().lower()
place_to_sleep = "Camp"
location = "Alaska"
holiday_price = 0
if budget <= 1000:
    if season == 'winter':
        location = "Morocco"
        holiday_price = budget * 0.45
    else:
        holiday_price = budget * 0.65
elif 1000 < budget <= 3000:
    place_to_sleep = "Hut"
    if season == 'winter':
        location = "Morocco"
        holiday_price = budget * 0.60
    else:
        holiday_price = budget * 0.80
else:
    place_to_sleep = "Hotel"
    holiday_price = budget * 0.9
    if season == 'winter':
        location = "Morocco"
print(f"{location} - {place_to_sleep} - {holiday_price:.2f}")

File: JS Basics/03 Nested Conditional Statements - More Exercises/05. Vacation.js
function holiday(budget, season) {
    budget = Number(budget);
    let price = 0;
    let location = "";
    let type = "";
    if (1000 >= budget) {
        type = "Camp";
        if (season == "Summer") {
            price = budget * 0.65;
            location = "Alaska";
        }
        else {
            price = budget * 0.45;
            location = "Morocco";
        }
    }
    else if (1000 < budget && budget <= 3000) {
        type = "Hut";
        if (season == "Summer") {
            price = budget * 0.80;
            location = "Alaska";
        }
        else {
            price = budget * 0.60;
            location = "Morocco";
        }
    }
    else {
        type = "Hotel";
        price = budget * 0.90;
        if (season == "Summer") {
            location = "Alaska";
        }
        else {
            location = "Morocco";
        }
    }
    console.log(`${location} - ${type} - ${price.toFixed(2)}`)
}
holiday(5000, "Winter")